components/experimental/src/units/ratio.rs

```diff
@@ -32,10 +34,20 @@ impl IcuRatio {
         self.0.numer()
     }
 
+    pub fn numerator_str(&self) -> String {
+        self.0.numer().to_string()
```

Comment:
issue: we generally don't want to use `to_string`, and with numbers especially the `to_string` implementation is not localized

Reply:
Suggestion: remove these two functions `numerator_str` and `denominator_str` until you need them. It seems easy enough to use `.numerator().to_string()` at the call site if you _really_ end up needing that (such as in datagen).